docs/versioned/tables/configuration.md

```diff
@@ -75,7 +96,6 @@ Iceberg tables support table properties to configure table behavior, like the de
 | commit.manifest-merge.enabled      | true             | Controls whether to automatically merge manifests on writes   |
 | history.expire.max-snapshot-age-ms | 432000000 (5 days) | Default max age of snapshots to keep while expiring snapshots    |
 | history.expire.min-snapshots-to-keep | 1                | Default min number of snapshots to keep while expiring snapshots |
-| history.expire.max-ref-age-ms      | `Long.MAX_VALUE` (forever) | For snapshot references except the `main` branch, default max age of snapshot references to keep while expiring snapshots. The `main` branch never expires. |
```

Comment:
should not be removed

Reply:
but I didn't find any usage of this parameter，is it deprecated?

Reply:
No, it isn't deprecated. We're in the process of adding it.